spec/stoplight/default_spec.rb

```diff
@@ -28,10 +28,6 @@
       expect(Stoplight::Default::BLACKLISTED_ERRORS).to be_an(Array)
     end
 
-    it 'is empty' do
-      expect(Stoplight::Default::BLACKLISTED_ERRORS).to be_empty
-    end
```

Comment:
This spec should check `WHITELISTED_ERRORS` now. And the "contains exception classes" spec from the whitelisted errors should be run against the blacklisted errors instead.